kahuna/public/js/components/gr-image-metadata/gr-image-metadata.html

```diff
@@ -280,7 +280,7 @@
 
                         <span class="metadata-line__info" ng-if="!ctrl.hasMultipleValues(ctrl.rawMetadata.byline)">
                             <span ng-if="ctrl.metadata.byline">
-                                <a ui-sref="search.results({query: (ctrl.metadata.byline | queryFilter:'by')})"
+                                <a ui-sref="search.results({query: (ctrl.metadata.byline | queryFilter:'by'), nonFree: ctrl.srefNonfree()})"
```

Comment:
thanks! there's still a few state transitions which don't have this because they're in other template files - I've seen this in at least labels and keywords. I think the important ones are gr-photoshoot.html, list-editor-compact.html and list-editor-info-panel.html, but you can find a full list by searching the repo for html files containing "search.results"